.gitignore

```diff
@@ -4,3 +4,7 @@
 /laminas-mkdoc-theme/
 /.phpunit.result.cache
 /composer.lock
+
+# Integration testings
+integration/*/vendor
```

Comment:
probably add `/` prefix like the above list to ensure directory `integration` that ignored in the root directory.